### PR TITLE
Add Router and add basic styling in all project

### PR DESCRIPTION
**Pull request by francksefu**
Add your own styles to improve the look and feel. -> done
For every page, you should create a <Route>. -> done
Make sure that navigation links for all 3 routes are displayed on each page. -> done
Use [React Router](https://reactrouter.com/) to make the routes work. -> done

### CODE REVIEW

File: .eslintignore [hunk 1]
```diff
@@ -1,4 +1,5 @@
 src/App.js
 src/index.js
 src/reportWebVitals.js
-src/components/Calculator.js
\ No newline at end of file
+src/components/Calculator.js
+src/components/Home.js
```
Comment (emmiiorji) [inline .eslintignore]: Kindly delete this file as it aims to defeat the purpose of having linters set up. Linters represent coding standards that you share with your team and attempts should not be made to bypass its checks.

**Review by emmiiorji — CHANGES_REQUESTED** [review]
Hi @francksefu, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

<div align='center'>

</div>

You have submitted a project with configuration that seeks to disable linter checks. Kindly delete the `.eslintignore` file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/Home.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+function Home() {
+  return (
+    <div className="home">
```
Comment (ShadyShawkat) [inline src/components/Home.js:3]: - [ ] Please remove the "dots" before every nav link to stick to the given design.
image

**Review by ShadyShawkat — CHANGES_REQUESTED** [review]
## Hi @francksefu, 

Good job so far! 👍🏻 
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight
- All linters are passing ✔️ 
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 

Image

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — APPROVED** [review]
Hi @francksefu , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**